Sources/AutoLayout/ConstrainableProxy.swift

```diff
@@ -844,7 +867,8 @@ public extension BaselineConstrainableProxy {
     }
 }
 
-public protocol PositionConstrainableProxy: EdgesConstrainableProxy, SizeConstrainableProxy, CenterConstrainableProxy {}
+public protocol PositionConstrainableProxy:
+    EdgesConstrainableProxy, SizeConstrainableProxy, AspectRatioConstrainableProxy, CenterConstrainableProxy {}
```

Comment:
shouldn't `AspectRatioConstrainableProxy` be a part of `SizeConstrainableProxy` instead, since it effectively only affects size? (and not position)

One could argue the `aspectRatio` could even be just one more operator provided by `SizeConstrainableProxy`, instead of creating `AspectRatioConstrainableProxy` just for it 🤔

Reply:
I had it originally on `SizeConstrainableProxy` but thought it might seem out of place. Will put it back there, thanks!